staging/src/k8s.io/dynamic-resource-allocation/structured/allocator.go

```diff
@@ -1315,3 +1326,21 @@ func containsNodeSelectorRequirement(requirements []v1.NodeSelectorRequirement,
 	}
 	return false
 }
+
+type bitfield int32
+
+func (b bitfield) set(i int) bitfield {
```

Comment:
should `i` be of type int32 as well as the `bitfield`? The same applies to the `clear` function below.